### PR TITLE
[LES-514] Add lesson image size for lesson 

### PR DESCRIPTION
**Pull request by hannrywislou**

### CODE REVIEW

File: app/controllers/lessons_controller.rb [hunk 1]
```diff
@@ -22,10 +23,13 @@ def show
 
   def new
     @lesson = Lesson.new
+    image_params
+    set_image_link
```
Comment (evrusha) [inline app/controllers/lessons_controller.rb:27]: Why we need this in 'new' action?

File: app/controllers/lessons_controller.rb [hunk 2]
```diff
@@ -82,6 +86,11 @@ def set_video_link
     @lesson.save!
   end
 
+  def image_params
+    @image_name = @lesson.image&.filename
+    @image_size = @lesson.image&.byte_size
```
Comment (evrusha) [inline app/controllers/lessons_controller.rb:91]: Maybe we need to add it to 'show' view?

File: app/controllers/lessons_controller.rb (hunk 1 above)
Comment (hannrywislou) [reply app/controllers/lessons_controller.rb]: I already fix it.

**Review by Terapool — APPROVED** [review]

File: app/controllers/lessons_controller.rb [hunk 3]
```diff
@@ -26,6 +27,7 @@ def new
 
   def create
     @lesson = Lesson.new(lesson_params)
+    image_params
```
Comment (evrusha) [inline app/controllers/lessons_controller.rb:30]: Where is this method defined? Maybe `lesson_image_params`?

File: app/controllers/lessons_controller.rb [hunk 4]
```diff
@@ -82,6 +84,11 @@ def set_video_link
     @lesson.save!
   end
 
+  def lesson_image_params
+    @image_name = @lesson.image&.filename
+    @image_size = @lesson.image&.byte_size
```
Comment (evrusha) [inline app/controllers/lessons_controller.rb:89]: Why `@lesson.image&` and not `@lesson.lesson_image&`?

File: app/views/lessons/index.jbuilder [hunk 5]
```diff
@@ -9,4 +9,6 @@ json.pagy_metadata do
   json.page @pagy.page
   json.per_page @pagy.items
   json.count_pages @pagy.pages
+  json.image_size lesson.image.byte_size
+  json.image_name lesson.image.filename
```
Comment (evrusha) [inline app/views/lessons/index.jbuilder:13]: Where is `lesson` local variable defined?

**Review by evrusha — CHANGES_REQUESTED** [review]

**Review by evrusha — APPROVED** [review]

**Review by ritamma — APPROVED** [review]

File: app/controllers/lessons_controller.rb [hunk 6]
```diff
@@ -82,6 +85,11 @@ def set_video_link
     @lesson.save!
   end
 
+  def lesson_image_params
```
Comment (fooooxmr) [inline app/controllers/lessons_controller.rb:88]: It's better to name this method set_lesson_image_params

**Review by fooooxmr — APPROVED** [review]

File: app/controllers/lessons_controller.rb (hunk 6 above)
Comment (hannrywislou) [reply app/controllers/lessons_controller.rb:88]: Ok.

**Pull request merged**